src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "AvenirBold";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: 300;
+}
+
+body {
+  margin-top: 0;
+}
+
+.largeSearchBar {
+  position: relative;
+  margin: 20px 0;
+}
+
+.icon {
+  position: absolute;
+  width: 19px;
+  bottom: 26px;
+  left: 26px;
+}
+
+.inputLarge {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  color: #3d4e61;
+  line-height: 22px;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+}
+
+.inputLarge::placeholder {
+  font-family: "Avenir", sans-serif;
+}
+
+.inputLarge:hover {
+  box-shadow: 0 3px 8px rgba(61,78,97,0.2);
+}
+
+.inputLarge:focus {
+  outline: none;
+  font-family: "AvenirBold", sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.smallSearchBar {
+  position: relative;
+}
+
+.iconSmall {
+  width: 11px;
+  left: 13px;
+  bottom: 16px;
+}
+
+.inputSmall {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61,78,97,0.1);
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 14px;
+  color: #3d4e61;
+  line-height: 19px;
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+}
+
+.inputSmall::placeholder {
+  font-family: "Avenir", sans-serif;
+}
+
+.inputSmall:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.inputSmall:focus {
+  outline: none;
+  font-family: "AvenirBold", sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
```

Comment:
You duplicate a bunch of styles. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2nd item